_episodes_rmd/02-r-plotting.Rmd

```diff
@@ -48,7 +54,101 @@ library(tidyverse)
 gapminder_1997 <- read_csv("data/gapminder_1997.csv")
 ```
 
-# Creating our first plot
+# Loading and reviewing data
+
+> ## The tidyverse vs Base R
+> If you've used R before, you may have learned commands that are different than the ones we will be using during this workshop. We will be focusing on functions from the [tidyverse](https://www.tidyverse.org/). The "tidyverse" is a collection of R packages that have been designed to work well together and offer many convenient features that do not come with a fresh install of R (aka "base R"). These packages are very popular and have a lot of developer support including many staff members from RStudio. These functions generally help you to write code that is easier to read and maintain. We believe learning these tools will help you become more productive more quickly.
+{: .callout}
+
+First, we're going to load the tidyverse, which contains a set of useful functions that makes it easier for us to do complex analyses and create professional visualizations in R. The way we access alk these useful functions functions is by running the following command:
```

Comment:
Can we expand this a little bit to also explain what a package is? Here is what I'm thinking (feel free to edit as needed and note I also fixed a typo alk -> all):

First, we're going to load the tidyverse package. Packages are useful because they contain pre-made functions to do specific tasks. Tidyverse contains a set of functions that makes it easier for us to do complex analyses and create professional visualizations in R. The way we access all of these useful functions functions is by running the following command: